### PR TITLE
app.js: Use encodeURIComponent for project urls

### PR DESCRIPTION
**Pull request by hemangsk**
Fixes https://github.com/coala/projects/issues/150

Also add arrows for project navigation
screen shot 2017-01-28 at 2 07 31 pm

### CODE REVIEW

**Comment by sils** [comment]
how does it look on mobile/with a smaller screen?

**Comment by sils** [comment]
so... how does it look?

**Comment by hemangsk** [comment]
screen shot 2017-01-28 at 10 09 40 pm

File: resources/js/app.js [hunk 1]
```diff
@@ -62,11 +62,11 @@
 				$scope.projects_url_dict = {}
 				
 				angular.forEach($scope.projectList, function(value, key){
-					value["url"] = value["name"].split(' ').join('_').toLowerCase();
+					value["url"] = encodeURIComponent(value["name"].split(' ').join('_').toLowerCase());
```
Comment (sils) [inline resources/js/app.js]: with that, do you still need the plit/join?

Comment (hemangsk) [inline resources/js/app.js]: Yeah actually it replaced spaces with %20, so links won't be easy to type directly, like improve%20diff%20handling, after split/join its like improve_diff_handling

**Comment by sils** [comment]
ack https://github.com/coala/projects/pull/151/commits/cc5a37869e20be622e56878545263cd791dad93b

File: index.html [hunk 2]
```diff
@@ -26,7 +26,7 @@
     <link href="https://fonts.googleapis.com/css?family=Roboto+Mono:300,300i,400,400i,700,700i|Roboto:100,200,300,400,500,600,700,800,900|Ubuntu+Mono|Overpass+Mono|Inconsolata" rel="stylesheet">
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
 
-    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
```
Comment (sils) [inline index.html]: why

File: partials/tabs/projects.html [hunk 3]
```diff
@@ -91,6 +91,15 @@
...
 
       </div>
+
+ </div>
+<div ng-show="currentProject" class="arrows">
+  <div class="row">
+    <div class="col-md-6 center"><i ng-click="moveToNext('left')" class="fa fa-arrow-circle-left arrow-click" aria-hidden="true"></i>
+</div>
+<div class="col-md-6 center"><i ng-click="moveToNext('right')" class="fa fa-arrow-circle-right arrow-click" aria-hidden="true"></i>
+</div>
```
Comment (sils) [inline partials/tabs/projects.html]: indentation improper

File: index.html (hunk 2 above)
Comment (hemangsk) [inline index.html]: It was slow

File: resources/js/app.js (hunk 1 above)
Comment (sils) [inline resources/js/app.js]: ok

File: index.html (hunk 2 above)
Comment (sils) [inline index.html]: why is n't this happening in an own commit and why don't you change the others as well then

**Comment by hemangsk** [comment]
reack 648e833

**Comment by gkrishnan724** [comment]
ack   2b8bfae c9db117  	6b66a0b  b33bb61  648e833

**Comment by hemangsk** [comment]
@rultor merge

**Pull request merged**